station-datafixer-v0/src/main/java/net/modificationstation/stationapi/api/datafixer/DataFixers.java

```diff
@@ -71,6 +74,23 @@ public static <T> boolean requiresUpdating(Dynamic<T> dynamic) {
         return false;
     }
 
+    public static <T> List<Triple<ModID, Integer, Integer>> getUpdateList(Dynamic<T> dynamic) {
```

Comment:
A custom record class (UpdateData?) would be nicer because no autoboxing.
Also, a HashSet would probably make more sense, maybe with a return type of Collection, since there's no need to allow duplicate elements and indexed access.